feature/gribi/ate_tests/route_removal_non_default_vrf/README.md

```diff
@@ -0,0 +1,56 @@
+# TE-6.2: Route Removal In Non Default VRF
```

Comment:
The folder name need to end in _test so we know it's a test package.

`feature/gribi/ate_tests/route_removal_non_default_vrf_test`

Reply:
Done. Thanks!